sdk/core/azure-core/setup.py

```diff
@@ -69,7 +69,8 @@
     },
     python_requires=">=3.7",
     install_requires=[
-        "requests>=2.18.4",
+        "anyio>=3.0,<5.0",
+        "requests>=2.21.0",
```

Comment:
Out of curiosity, why do we need to bump the floor for `requests`?

Reply:
With the introduction of anyio, the `mindependency` job now fails if I don't bump `requests` due to:

```
ERROR: Cannot install anyio==3.0.0 and requests==2.19.0 because these package versions have conflicting dependencies.

The conflict is caused by:
    anyio 3.0.0 depends on idna>=2.8
    requests 2.19.0 depends on idna<2.8 and >=2.5
```